Wikipedia/Code/WMFURLCache.m

```diff
@@ -28,9 +28,16 @@ - (NSCachedURLResponse *)cachedResponseForRequest:(NSURLRequest *)request {
         }
     }
     
+    dispatch_block_t done = ^{
+        if (!response) {
+            wmf_postNetworkRequestBeganNotification(request.HTTPMethod, request.URL.absoluteString);
```

Comment:
looks good to me - maybe have the response passed to the block instead of a captured var. Does this capture the value when done is assigned? I thought it would require `__block` to be updated after the block creation.